Description: prevent breaking source tree
 Without this patch abiword breaks its own source tree
 so the second build fail
Author: Dmitry Smirnov <onlyjob@member.fsf.org>
Forwarded: not-needed
Last-Update: 2020-06-05
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/Makefile.am
+++ b/Makefile.am
@@ -64,11 +64,7 @@
 	lt-compile-resource
 
 DISTCLEANFILES = \
-	abiword-$(ABIWORD_SERIES).pc \
-	plugin-builtin.m4 \
-	plugin-configure.m4 \
-	plugin-list.m4 \
-	plugin-makefiles.m4
+	abiword-$(ABIWORD_SERIES).pc
 
 abiword-$(ABIWORD_SERIES).pc: libabiword.pc
 	cp $< $@
--- a/src/wp/test/Makefile.am
+++ b/src/wp/test/Makefile.am
@@ -19,4 +19,4 @@
 #include $(top_srcdir)/includes.mk
 
 SUBDIRS= xp @PLATFORM@
-DIST_SUBDIRS = cocoa unix win xp
+DIST_SUBDIRS = unix xp
